app/helpers/application_helper.rb

```diff
@@ -1,2 +1,27 @@
 module ApplicationHelper
+
+  def categories
+    Category.all
+  end
+
+  def item_status_check(item)
+    if item.active?
+      button_to 'Add to Cart', carts_path(item_id: item.id)
+    else
+      button_to 'Item Retired', retired_item_path(item_id: item.id)
+    end
+  end
```

Comment:
If this is only used within the item views, I would throw this in it's own ItemHelper. Not sure if that is the case or not here, but something to think about for the future :)